chapters/annotations.tex

```diff
@@ -2324,6 +2324,19 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
+
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license text file for the class.
+The annotation may give a scalar value or an array of values.
+The license text files are human-readable files containing the license conditions.
+If the annotation specifies an array of license text files the conditions in all of them must be satisfied (i.e., it is not intended for dual-licensing).
+External functions may also contain this annotation (with similar content), see \cref{annotations-for-external-libraries-and-include-files}.
+
+When generating a distributable version for a translated model the license text files for used functions and top-level classes should be included.
```

Comment:
```suggestion
When generating a distributable version (e.g., an FMU) for a translated model the license text files for used functions and top-level classes should be included.
```
As suggested.

Reply:
@eshmoylova - your comment above.

Reply:
That would work for me.

Reply:
Is FMU already declared/defined in the MoLang Spec?

Reply:
@eshmoylova  still ok?

Reply:
FMU is not defined or used so far. But we refer to C89 and C99, for example, without providing any explanation or links. So, it should be acceptable to refer to FMU like that as well. It might be nicer to make it a link.

Reply:
Yes, that was the intention of my comment: FMU should be declared and not considered as given.

Reply:
If linked, I think it should be handled similar to gettext; as a citation with the link given in the bibliography.

Reply:
We can add the link later, if needed, but is the current proposed text good enough for the moment?

Reply:
Looks good.